### DISCUSSION TITLE
Is there a way to tree shake locales?

### DISCUSSION THREAD

**Discussion by gfohl** [Azure/communication-ui-library#3586] [Q&A]
It looks like all locales are being bundled:
<img width="670" alt="image" src="https://github.com/Azure/communication-ui-library/assets/67913063/2be85f9d-cf93-44a1-9fd1-7b90d8a01aec">

Provider looks like this:
<img width="373" alt="image" src="https://github.com/Azure/communication-ui-library/assets/67913063/c383e515-ccfa-4a7c-92bf-4be0f195e48d">

**Comment by prabhjot-msft** [comment]
@gfohl thanks for initiating this discussion, we are looking into it. We have created an issue https://github.com/Azure/communication-ui-library/issues/3600 and will keep you posted.

**Comment by prabhjot-msft** [comment]
@gfohl Thanks for highlighting the issue, a PR has been merged into main branch https://github.com/Azure/communication-ui-library/pull/3615 to fix this. Please wait for next version release to get the fix, closing the discussion. Please feel free to reopen incase there is any concern.

**Comment by gfohl** [comment]
@prabhjot-msft thanks so much for putting in a fix, looks great! The last release was a couple weeks ago, can we expect another this month?
